lib/ctblmono.gi

```diff
@@ -2176,6 +2176,74 @@ InstallMethod( IsMinimalNonmonomial,
     return false;
 end );
 
+InstallMethod( IsMinimalNonmonomial,
+    "for a non-solvable group",
+    [ IsGroup ],
+    function( K )
+
+    local info, q, p, f;
+
+    if IsSolvableGroup(K) then
+      TryNextMethod();
+    fi;
+
+    # A non-solvable group all of whose proper quotients are monomial
+    # must have every proper quotient solvable. Hence G must be simple.
+    if not IsSimpleGroup(K) then
+      return false;
+    fi;
+
+    # If in addition, every proper subgroup is monomial, then every
+    # proper subgroup must be solvable, so G must be minimal simple.
+    #
+    # A minimal non-simple group must be (by Thompson's N-Groups paper):
```

Comment:
It is not hard to google it, but perhaps we could still insert e.g. the DO Ito make it absolutely clear?
```suggestion
    # A minimal non-simple group must be (by Corollary 1 in Thompson's 1968
    #  N-Groups paper, doi:10.1090/S0002-9904-1968-11953-6):
```